packages/react-instantsearch-hooks-server/rollup.config.js

```diff
@@ -39,14 +39,15 @@ const plugins = [
 
 const createConfiguration = ({ name, minify = false } = {}) => ({
   input: 'src/index.ts',
-  external: ['react', 'react-dom', 'react-instantsearch-hooks'],
+  external: ['react', 'react-instantsearch-hooks'],
+  inlineDynamicImports: true,
   output: {
     file: `dist/umd/ReactInstantSearch${name}${minify ? '.min' : ''}.js`,
     name: `ReactInstantSearch${name}`,
     format: 'umd',
     globals: {
       react: 'React',
-      'react-dom': 'ReactDOM',
```

Comment:
Can you explain this change? (And if it makes sense, add a comment.)

Reply:
the file is deleted in https://github.com/algolia/react-instantsearch/pull/3465, but it isn't used as we don't import react-dom, only react-dom/server

Reply:
Shouldn't we name it `ReactDOMServer` then?